src/js/events/ChaosEvents.js

```diff
@@ -0,0 +1,6 @@
+var ChaosEvents = {
+  FINISHED: "CHAOS_EVENTS_FINISHED",
+  STARTED: "CHAOS_EVENTS_STARTED"
```

Comment:
Fair enough, although what I was trying to say is that we must keep in mind we can't currently display an actual "chaos function is running" state.